src/template_compiler.erl

```diff
@@ -454,10 +454,10 @@ expand_translation(Token, _Runtime, _Context) ->
 extract_translations(Tokens) ->
     lists:foldl(
         fun
-            ({trans_text, _, Text}, Acc) ->
-                [ template_compiler_utils:unescape_string_literal(Text) | Acc ];
-            ({trans_literal, _, Text}, Acc) ->
-                [ template_compiler_utils:unescape_string_literal(Text) | Acc ];
+            ({trans_text, LineAndNumber, Text}, Acc) ->
+                [ {template_compiler_utils:unescape_string_literal(Text), [], [LineAndNumber]} | Acc ];
+            ({trans_literal, LineAndNumber, Text}, Acc) ->
+                [ {template_compiler_utils:unescape_string_literal(Text), [], [LineAndNumber]} | Acc ];
```

Comment:
That is the line and column number, it is normally used in the generated .pot file for the source reference.